### DISCUSSION TITLE
Welcome to Graphin Discussions!

### DISCUSSION THREAD

**Discussion by pomelo-nwu** [antvis/Graphin#337] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cliffordfajardo** [comment]
Hi Graphin Community 👋 
I've been using the library for about 2 weeks & I've been enjoying it a lot!

Before Graphin, my team was using Cisco's old NextUI library to render network topologies & help us our infrastructure. NextUI unfortunately is no longer maintained, the codebase is 2015 style code and it was hard to develop & maintain. 

Fortunately after about a month of researching I stumbled upon Graphin

What I like about Graphin
- options for rendering to canvas or svg. It can handle thousands of nodes easily
- can use JSX elements (or custom DOM) to render tooltips for canvas elements!
- codebase is in typescript & is modern
- has plugins & features like: [combo's](https://g6.antv.vision/en/docs/manual/middle/elements/combos/defaultCombo) feature, [timebar](https://g6.antv.vision/en/docs/design/component/timebar#gatsby-focus-wrapper),
- API to create custom layouts and behavior

I'm still new to Graphin, but I hope to contribute back and help build the Graphin community & its usage in the United States.

Big shout out to @pomelo-nwu , @zxc0328, @baizn, @timlrx , @CodesAreHonest @Blakko  and the rest of the Graphin maintainer & [contributors](https://github.com/antvis/Graphin/graphs/contributors) on this great library


<br>

## Tips for Contributing to the community 👥
When reporting a bug or issue, it alway's helps if you can be detailed yet concise in your description's & have a reproducible code example. **Also, please imagine yourself in the place of an open source maintainer when asking for help or contributing**
- open source maintainer's contribute out of the goodness of their heart 💙 in their free time after their typical work & it's unpaid!
- open source is hard
    - Video [The Hard Parts of Open Source by Evan Czaplicki](https://www.youtube.com/watch?v=o_4EX4dPppA), 
    - Article: [Open Source is hard by Jose Gonzalez](http://josediazgonzalez.com/2016/01/26/open-source-is-hard/)
    - Video: [Why you should contribute to open source by Marit van Dijk](https://www.youtube.com/watch?v=GAqfMNB-YBU)

For reproducible code examples, I recommend [stackblitz](https://stackblitz.com/) (which is VSCode in the browser):
   -  it allows you to create a fast sample of your code
   - allows you to drag files from your computer to the stackblitz editor running in your web browser
   - This is an example of Graphin code sample created on Stackblitz
      - Code: [stackblitz.com/edit/react-ts-ma2vpb](stackblitz.com/edit/react-ts-ma2vpb)
      - Demo: [react-ts-ma2vpb.stackblitz.io](react-ts-ma2vpb.stackblitz.io)


Let's have fun & let's us continue making this an amazing & welcoming community 🎉
